extension/pages/settings/settings.html

```diff
@@ -716,6 +716,10 @@ <h2>
 							<label for="events-easterEggs">Let you know when an easter egg shows.</label>
 							<label for="events-easterEggs" class="tabbed note">Detection scripts supposedly lower your egg rate, use at your own risk.</label>
 						</div>
+						<div class="option">
+							<input id="events-eventWorth" type="checkbox" />
+							<label for="events-eventWorth">Show event worth.</label>

```

Comment:
It should be more clear what it actually does.

Reply:
Neither is it an event. Events are like the easter eggs, etc..., not the event page.